src/pages/about.js

```diff
@@ -1,66 +1,118 @@
+/* eslint-disable react/prop-types */
 import React from 'react';
-
+import { graphql } from 'gatsby';
 import Layout from '../components/layout';
 import SEO from '../components/seo';
+import Img from 'gatsby-image';
+import LearnTeachCoding from '../assets/Learn-Teach-Coding.svg';
+
+function AboutPage(props) {
+  // const { avatarSethAlexanderImage } = [
+  //   data.avatarSethAlexanderImage.childImageSharp.fluid,
+  // ];
+  console.log(props.data);
 
-function AboutPage() {
   return (
     <Layout>
-      <SEO keywords={[`nashville`, `freecodecamp`]} title="Home" />
-
-      <section className="text-center">
-        <h2 className="inline-block p-3 mb-4 text-2xl font-bold bg-yellow-400">
-          Code of Conduct
-        </h2>
+      <SEO keywords={[`nashville`, `freecodecamp`]} title="About" />
```

Comment:
So it looks like line 17 achieves what `meta` tags in an `HTML` file's `head` achieve. You just have to place them inside the `<Layout>`, of the `return` parameters of the `function`, sort of like so.

<img width="409" alt="Screen Shot 2021-05-10 at 5 24 37 PM" src="https://user-images.githubusercontent.com/66753884/117731961-9bbcbf80-b1b4-11eb-875a-347290b6f79b.png">

Reply:
It's because we are using React Helmet. React Helmet will dynamically add those tags as meta tag inside the head tags when rendering the page.